src/mobile/src/ui/components/AnimatedComponent.js

```diff
@@ -145,6 +145,10 @@ class AnimatedComponent extends Component {
                 case 'slideOutRightSmall':
                     animatedStyle = merge({}, animatedStyle, { width, transform: [{ translateX: this.slideValue }] });
                     break;
+                case 'slideInBottom':
+                case 'slideOutBottom':
+                    animatedStyle = merge({}, animatedStyle, { width, transform: [{ translateY: this.slideValue }] });
```

Comment:
Since `animatedStyle` is defined as inside this function and as a new object a deep merge can be avoided here and we can just reassign to `animatedStyle`